src/Tribe/Log/README.md.bak

```diff
@@ -0,0 +1,152 @@
+# Monolog-based logging
```

Comment:
I think this file was an accidental addition to the PR.

Reply:
You never backup too much.